lrpar/src/lib/ctbuilder.rs

```diff
@@ -52,6 +54,30 @@ struct CTConflictsError<StorageT: Eq + Hash> {
     stable: StateTable<StorageT>,
 }
 
+/* Quote by default quotes option values as an empty string for `None`,
+ * and the value itself for `Some`, where here we actually want the
+ * Some/None included.
+ *
+ * See: https://github.com/dtolnay/quote/issues/20 */
+struct QuoteOption<T>(Option<T>);
+
+impl<T: ToTokens> ToTokens for QuoteOption<T> {
+    fn to_tokens(&self, tokens: &mut TokenStream) {
+        tokens.append_all(match self.0 {
+            Some(ref t) => quote! { ::std::option::Option::Some(#t) },
+            None => quote! { ::std::option::Option::None },
+        });
+    }
+}
+
+struct UnsuffixedUsize(usize);
```

Comment:
I like the idea behind this: it'll need a motivational docstring to remind us *why* it's here though!

Reply:
Should be fixed in [f6d33f4](https://github.com/softdevteam/grmtools/pull/494/commits/f6d33f49b5d685aa9952122f6640f80433926fdc)